server/routerlicious/packages/services/src/secretManager.ts

```diff
@@ -10,6 +10,10 @@ import * as core from "@fluidframework/server-services-core";
  * Users requiring encryption of secrets are expected to have their own implementation of ISecretManager.
  */
 export class SecretManager implements core.ISecretManager {
+	public getIncomingKeyVersion(): core.EncryptionKeyVersion {
+		return undefined;
+	}
+
 	public decryptSecret(encryptedSecret: string): string {
```

Comment:
let's add the second param "encryptionKeyVersion?: EncryptionKeyVersion" to these functions for consistency?

Reply:
Good catch, will do.

Reply:
Added.